samples/node/pub_sub_electron_node/README.md

```diff
@@ -62,64 +60,105 @@ Replace with the following with the data from your AWS account:
 
-2. Setup `settings.ts`. You will need setup the `region` and `endpoint` in `settings.ts` to setup the e
+2. Setup `node/pub_sub_electron_node/settings.ts`. You will need setup the `region` and `endpoint` in the setting file.
 
 3. Install node packages
 ```sh
-npm install
+npm install .
 ```
 
 4. Build and Run
 ```sh
-npm run build
 npm run start
 ```
 
-## Electron Q&A
-### Warning: `objc[79765]: Class WebSwapCGLLayer is implemented in both ` ?
+## Package
+Please refer to (Electron-tutorial-packaging)[https://www.electronjs.org/docs/latest/tutorial/tutorial-packaging] for packaging details.
+As our sample is using typescript, you would need include the compiled js files while packaging. You can config the `package.json` or `forge.config.js` to set it.
+Example `package.json`:
+```js
+"build": {
+    "files": [
+      "dist/*"
+    ]
+  }
+```
 
-This is an issue running Electron on MacOS. The API has a name duplication for "WebSwapCGLLayer". The warning should not affect your development. The issue is fixed by Electron in v22. Unfortunately, our SDK currently only supports Electron@19 and below.
 
-More info: https://github.com/electron/electron/issues/33685
+## Electron Q&A
+### Warning: `objc[79765]: Class WebSwapCGLLayer is implemented in both ... `
+This is an issue running Electron on MacOS. The API has a name duplication for "WebSwapCGLLayer". The warning should not affect your development. The issue is fixed by Electron in v22.
 
-### How to debug with Dev Tools
-You can open dev tool using the following API:
-```
-  win.webContents.openDevTools()
-```
+More info: https://github.com/electron/electron/issues/33685
 
 ### SyntaxError: Unexpected token '?'
 Please check your dependency and Node version. If the error is not from your code, it is most likely your dependency is using a different version of node. As the nullish coalescing operator (??) is introduced in Node14, using Node14+ would help.
 
 ### N-API call failed: napi_create_external_arraybuffer( env, data_buffer->buffer, data_buffer->len, s_finalize_external_binary_byte_buf, data_buffer, &napi_binary).
-Electron removed support for `napi_create_external_arraybuffer` since Electron@20. You can find more information from the Electron community here: https://github.com/electron/electron/issues/35801. There is no solid solution for the issue right now. Our team is actively working on resolving it.
+The issue should be fixed in release v1.19.1.
+Electron removed support for `napi_create_external_arraybuffer` since Electron@20. You can find more information from the Electron community here: https://github.com/electron/electron/issues/35801.
```

Comment:
Trivial: "Electron removed support for `napi_create_external_arraybuffer` in Electron v20."